src/brainglobe_napari/atlas_viewer_widget.py

```diff
@@ -117,9 +118,8 @@ def _on_download_selected_atlas_clicked():
             if self._selected_atlas_row is not None:
                 if self._selected_atlas_name not in get_downloaded_atlases():
                     # instantiation will trigger download
-                    selected_atlas = BrainGlobeAtlas(  # noqa: F841
-                        self._selected_atlas_name
-                    )
+                    _ = BrainGlobeAtlas(self._selected_atlas_name)
```

Comment:
Is it worth using [`bg_atlasapi.update_atlases.install_atlas("atlas_name")`](https://github.com/brainglobe/bg-atlasapi/blob/b6e955df1b88b6a998cb4653cc2d76ab57911126/bg_atlasapi/update_atlases.py#L57) here, just to be more explicit, and also allow us to update `install_atlas()` to download the atlas without the overhead of instantiating it?

Reply:
Definitely!